tests/jballerina-unit-test/src/test/resources/test-src/expressions/varref/record-variable-reference.bal

```diff
@@ -191,56 +191,188 @@ function testRestParameterType() returns boolean {
     map<anydata|error> other1 = {};
     map<any> other2 = {};
 
-    IntRestRecord rec1 = { name: "A", married: true, "age": 19, "token": 200 };
-    { name, ...other1 } = rec1;
+    IntRestRecord rec1 = {name: "A", married: true, "age": 19, "token": 200};
+    {name, ...other1} = rec1;
 
     any a1 = other1;
 
     return a1 is map<anydata|error>;
 }
 
-// TODO: Uncomment below tests once record literal is supported with var ref
+function testVarAssignmentOfRecordLiteral1() {
+    string fName;
+    boolean married;
+    int theAge;
+    string format;
 
-//function testVarAssignmentOfRecordLiteral() returns (string, boolean, int, string) {
-//    string fName;
-//    boolean married;
-//    int theAge;
-//    string format;
-//    map theMap;
```

Comment:
Why aren't we (fixing and) adding theMap also?

Reply:
it's handled in other test scenarios.